webview-ui/src/components/account/AccountView.tsx

```diff
@@ -118,8 +93,7 @@ export const ClineAccountView = () => {
 
 		const fetchUserData = async () => {
 			try {
-				await getUserCredits()
-				await getUserOrganizations()
+				Promise.all([getUserCredits(), getUserOrganizations()])
```

Comment:
Promise.all() does not guarantee order.  This was explicitly left as two awaited calls to maintain order of the calls so that a race condition was handled.